README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_get_data_DOM/)
+    - [DEMO LINK](https://GUSILLUS.github.io/js_get_data_DOM/)
```

Comment:
![image](https://user-images.githubusercontent.com/56752562/220553427-cb52f79b-edcc-4b8f-9832-54eb2174cc37.png)
Try to deploy project.

Reply:
Done